fleet_management_api/api_impl/controllers/route.py

```
@@ -181,20 +187,16 @@ def _create_empty_route_visualization(route_id: int) -> _Response:
-        return _text_response(
-            f"Empty route visualization (route ID={route_id}) has been created."
-        )
+        return _text_response(f"Empty route visualization (route ID={route_id}) has been created.")
 
 
-def _find_nonexistent_stops(*routes: _Route) -> _Response:
+def _find_nonexistent_stops(tenants: _AccessibleTenants, *routes: _Route) -> _Response:
     for route in routes:
         checked_id_set: set[int] = set(route.stop_ids)
         for id_ in checked_id_set:
             foo = lambda x, id_: x == id_
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `foo` as a `def`

(E731)